src/script_opt/FuncInfo.cc

```diff
@@ -0,0 +1,506 @@
+    "record_type_to_vector",
+    "remask_addr",
+    "remove_prefix",
+    "remove_suffix",
+    "rename",
+    "reverse",
+    "rfind_str",
+    "rjust",
+    "rmdir",
+    "rotate_file",
+    "rotate_file_by_name",
+    "routing0_data_to_addrs",
+    "rstrip",
+    "safe_shell_quote",
+    "same_object",
+    "sct_verify",
+    "set_buf",
+    "set_contents_file",
+    "set_current_conn_bytes_threshold",
+    "set_current_conn_duration_threshold",
+    "set_current_conn_packets_threshold",
+    "set_file_handle",
+    "set_inactivity_timeout",
+    "set_keys",
+    "set_login_state",
+    "set_network_time",
+    "set_record_packets",
+    "set_secret",
+    "set_ssl_established",
+    "setenv",
+    "sha1_hash",
+    "sha1_hash_finish",
+    "sha1_hash_init",
+    "sha1_hash_update",
+    "sha256_hash",
+    "sha256_hash_finish",
+    "sha256_hash_init",
+    "sha256_hash_update",
+    "skip_further_processing",
+    "skip_http_entity_data",
+    "skip_smtp_data",
+    "split_string",
+    "split_string1",
+    "split_string_all",
+    "split_string_n",
+    "sqrt",
+    "srand",
+    "starts_with",
+    "str_smith_waterman",
+    "str_split_indices",
+    "strcmp",
+    "strftime",
+    "string_cat",
+    "string_fill",
+    "string_to_ascii_hex",
+    "string_to_pattern",
+    "strip",
+    "strptime",
+    "strstr",
+    "sub",
+    "sub_bytes",
+    "subnet_to_addr",
+    "subnet_width",
+    "subst_string",
+    "suspend_processing",
+    "swap_case",
+    "syslog",
+    "system",
+    "system_env",
+    "table_keys",
+    "table_values",
+    "terminate",
+    "time_to_double",
+    "to_addr",
+    "to_count",
+    "to_double",
+    "to_int",
+    "to_json",
+    "to_lower",
+    "to_port",
+    "to_string_literal",
+    "to_subnet",
+    "to_title",
+    "to_upper",
+    "topk_add",
+    "topk_count",
+    "topk_epsilon",
+    "topk_get_top",
+    "topk_init",
+    "topk_merge",
+    "topk_merge_prune",
+    "topk_size",
+    "topk_sum",
+    "type_aliases",
+    "type_name",
+    "unescape_URI",
+    "uninstall_dst_addr_filter",
+    "uninstall_dst_net_filter",
+    "uninstall_src_addr_filter",
+    "uninstall_src_net_filter",
+    "unique_id",
+    "unique_id_from",
+    "unlink",
+    "uuid_to_string",
+    "val_footprint",
+    "write_file",
+    "x509_check_cert_hostname",
+    "x509_check_hostname",
+    "x509_from_der",
+    "x509_get_certificate_string",
+    "x509_issuer_name_hash",
+    "x509_ocsp_verify",
+    "x509_parse",
+    "x509_set_certificate_cache",
+    "x509_set_certificate_cache_hit_callback",
+    "x509_spki_hash",
+    "x509_subject_name_hash",
+    "x509_verify",
+    "zeek_args",
+    "zeek_is_terminating",
+    "zeek_version",
+    "zfill",
+};
+
+// Ones not listed:
+//
+// Broker::*
+//	These can manipulate unspecified (at script level) records.
+//
+// Cluster::publish_hrw
+// Cluster::publish_rr
+//	These call script functions to get topic names.
+//
+// Log::__write
+//	Calls log policy functions.
```

Comment:
The new `Log::__delay_finish()` invokes callbacks

Reply:
Thanks for flagging! (And I'm pleased to see that the new ZAM maintenance helper script indeed correctly flags it, and others, as requiring inspection.)

The nice thing about how the code deals with BiFs is new/unanalyzed ones are by default treated as "who knows what side effects this might have", so if it took me a while to get around to looking over `Log::__delay_finish()` there wouldn't have been any harm.